dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/info/BroadcastReceiverNetworkInfoProvider.kt

```diff
@@ -101,13 +101,40 @@ internal class BroadcastReceiverNetworkInfoProvider :
             NetworkInfo(
                 connectivity,
                 carrierName.toString(),
-                carrierId
+                carrierId,
+                cellularTechnology = getCellularTypeName(subtype)
```

Comment:
shouldn't you add the cellularTechnology also in the else branch ? Is this constrained by the API level ?